### PR TITLE
Peers monitoring tool

### LINKED ISSUE #1329: Tool to monitor peers.txt IP addresses
Create a tool (either a new cli command or a new `cmd/` binary) that takes in a list of peers (ip:ports, newline separated, skipping comments and empty lines (a function exists somewhere in the `daemon` or `pex` package to parse this format)).  The tool connects to each of the peers, waits for the introduction packet (or times out) and produces a report with the status of the peer (unreachable, reachable, sent_introduction, introduction_parameters).  Introduction_parameters were added in v0.25.0 so will be absent for earlier peer versions.

The initial purpose is to check the status of peers in https://downloads.skycoin.net/blockchain/peers.txt

### PR DESCRIPTION
**Pull request by nkryuchkov**
Fixes #1329 

Changes:
- Added new cmd tool `monitor-peers` to scan peers in the network

Does this change need to mentioned in CHANGELOG.md?
No

### CODE REVIEW

File: src/visor/config.go [hunk 1]
```diff
@@ -81,23 +81,23 @@ func (c Config) Verify() error {
 	}
 
 	if c.UnconfirmedVerifyTxn.BurnFactor < params.UserVerifyTxn.BurnFactor {
-		return fmt.Errorf("UnconfirmedVerifyTxn.BurnFactor must be >= params.UserVerifyTxn.BurnFactor (%d)", params.UserVerifyTxn.BurnFactor)
+		return fmt.Errorf("unconfirmedVerifyTxn.BurnFactor must be >= params.UserVerifyTxn.BurnFactor (%d)", params.UserVerifyTxn.BurnFactor)
```
Comment (gz-c) [inline src/visor/config.go]: Undo the capitalization changes of the intro params in log messages

File: src/daemon/messages.go [hunk 2]
```diff
@@ -362,7 +362,7 @@ func NewIntroductionMessage(mirror uint32, version int32, port uint16, pubkey ci
 func newIntroductionMessageExtra(pubkey cipher.PubKey, userAgent string, verifyParams params.VerifyTxn) []byte {
 	if len(userAgent) > useragent.MaxLen {
 		logger.WithFields(logrus.Fields{
-			"userAgent": userAgent,
+			"UserAgent": userAgent,
```
Comment (gz-c) [inline src/daemon/messages.go]: undo

File: src/visor/config.go (hunk 1 above)
Comment (nkryuchkov) [reply src/visor/config.go]: Done

File: src/daemon/messages.go (hunk 2 above)
Comment (nkryuchkov) [reply src/daemon/messages.go]: Done

**Pull request merged**